### ISSUE TITLE
Issue #15 — Baseline Learning Curves

### ISSUE THREAD

**Issue by christine-a11y** [christine-a11y/cassava-leaf-disease-classification#16]
Նպատակ

Հասկանալ model-ի training behavior-ը։

Tasks
 Plot training loss
 Plot validation loss
 Plot training accuracy
 Plot validation accuracy
 Ստուգել overfitting
 Ստուգել underfitting
Պետք է պատասխանենք

Model-ը սովորո՞ւմ է։

Overfitting կա՞։

Ո՞ր epoch-ից validation performance-ը վատանում է։

Deliverable

Learning curves + analysis։
